### ISSUE TITLE
AutoUpload parameter are shared by the two configured nextcloud servers

### ISSUE THREAD

**Issue by nextflo** [nextcloud/android#1280]
### Actual behaviour
I have 2 NC-Server, both are configured in the APP. AutoUpload parameters set in Configuration oft NC1 are also shown in Configuration oft NC2. The upload of new photos was done to the wrong one

### Expected behaviour
Do the upload to the one for which it is  configured. 
 
### Steps to reproduce
1. 
2. 
3. 

### Environment data
Android version:

Device model: 

Stock or customized system:

Nextcloud app version: Nextcloud Android-App Version 1.4.3

Nextcloud server version:

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by mario** [comment]
What version of the app?

**Comment by mario** [comment]
@AndyScherzinger could you check this out if this is the case in our branch?

**Comment by nextflo** [comment]
Nextcloud Android-App Version 1.4.3

**Comment by AndyScherzinger** [comment]
I just tested this on the branch and it works fine, no upload to the wrong server. So fixed with 1.5.0
